### PR TITLE
PR to address issues#36

### PR DESCRIPTION
**Pull request by RohitKumarJain**
This PR contains the fix for the issue raised for https://github.com/eclipse-ee4j/jaxb-tck/issues/36.

- After making the changes, I have tested the changes while triggering the CI job (https://ci.eclipse.org/jakartaee-tck/job/rjain-jaxb-tck/job/master/5/). The build is successful and all test cases are passing.

- Renamed JAXB-TCK acronym to XMLB-TCK.

- JAXB-TCK-3.0.jar installer is removed.

### CODE REVIEW

File: build.xml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    This program and the accompanying materials are made available under the
+    terms of the Eclipse Public License v. 2.0, which is available at
+    http://www.eclipse.org/legal/epl-2.0.
+    This Source Code may also be made available under the following Secondary
+    Licenses when the conditions for such availability set forth in the
+    Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+    version 2 with the GNU Classpath Exception, which is available at
+    https://www.gnu.org/software/classpath/license.html.
+    SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+-->
```
Comment (lukasj) [inline build.xml:12]: minor thing - let the header be formatted the same way as in other files (including empty lines)

**Review by alwin-joseph — APPROVED** [review]
LGTM

File: build.xml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+<project name="jaxb-tck" default="dist" basedir=".">
```
Comment (gurunrao) [inline build.xml:14]: jaxb-tck can be renamed to xmlb-tck

**Review by gurunrao — COMMENTED** [review]
https://ci.eclipse.org/jakartaee-tck/job/rjain-jaxb-tck/job/master/5/artifact/bundles/jakarta-xml-binding-tck-3.0.0.zip contains "?" folder at xml-binding-tck/XMLB-TCK-3.0

File: build.xml (hunk 1 above)
Comment (RohitKumarJain) [reply build.xml:18]: I have addressed and updated the header format.

Comment (lukasj) [reply build.xml:18]: thx!

File: build.xml (hunk 2 above)
Comment (RohitKumarJain) [reply build.xml]: done.

**Review by lukasj — APPROVED** [review]
LGTM, let @gurunrao comment wrt `?` be addressed in the other PR

**Comment by gurunrao** [comment]
@RohitKumarJain - "?" folder at xml-binding-tck/XMLB-TCK-3.0 can be addressed as a separate PR. thanks for addressing the https://github.com/eclipse-ee4j/jaxb-tck/issues/36.
CC @lukasj @alwin-joseph

**Pull request merged**